main.go

```diff
@@ -105,10 +106,23 @@ func main() {
 			Scheme:        mgr.GetScheme(),
 			EventRecorder: newEventRecorder(clientSet),
 		},
-		Log: ctrl.Log.WithName("controllers").WithName("iperf3")}).SetupWithManager(mgr); err != nil {
+		Log: ctrl.Log.WithName("controllers").WithName("iperf3"),
+	}).SetupWithManager(mgr); err != nil {
 		setupLog.Error(err, "unable to create controller", "controller", "Iperf3")
 		os.Exit(1)
 	}
+	if err = (&controllers.FioReconciler{
+		K8S: k8s.Access{
```

Comment:
this struct is shared between iperf and fio. could we please use a shared var in both cases?